webapp/app.py

```diff
@@ -13,18 +15,10 @@
     "certification.ubuntu.com",
     template_folder=templates_dir,
```

Comment:
Can we remove `L9-11` and make `template_folder="templates"`?

Reply:
obviously you meant `"../templates"` :P

Reply:
Actually, not unfortunately we can't until we fix https://github.com/canonical-web-and-design/canonicalwebteam.templatefinder/issues/2